### PR TITLE
More arousal logging; adds back aphro/nympho

### PR DESCRIPTION
**Pull request by Putnam3145**
## About The Pull Request

I was a goddamn idiot to just remove instead of adding better logging. Now all sources of arousal as well as climaxes are logged in the emote log, including whether they're forced or not.

## Why It's Good For The Game

Because the logging is good enough, we can bring back stuff like this without too much issue.

## Changelog
:cl:
add: (Hexa)crocin
add: (Hexa)camphor
add: Nymphomaniac quirk
admin: All climaxes and arousals are now logged, as well as genital exposure.
/:cl:

### CODE REVIEW

**Comment by ghost** [comment]
L

**Comment by moltoretardo** [comment]
L

File: code/modules/mob/living/carbon/human/species.dm [hunk 1]
```diff
@@ -1547,9 +1547,9 @@ GLOBAL_LIST_EMPTY(roundstart_race_names)
 		if(!user.UseStaminaBuffer(3, warn = TRUE))
 			return FALSE
 		user.do_attack_animation(target, ATTACK_EFFECT_ASS_SLAP)
-		target.adjust_arousal(20,maso = TRUE)
+		target.adjust_arousal(20,"masochism", maso = TRUE)
 		if (ishuman(target) && HAS_TRAIT(target, TRAIT_MASO) && target.has_dna() && prob(10))
-			target.mob_climax(forced_climax=TRUE)
+			target.mob_climax(forced_climax=TRUE, "masochism")
```
Comment (LetterN) [inline code/modules/mob/living/carbon/human/species.dm]:
```suggestion
			target.mob_climax(forced_climax=TRUE, cause = "masochism")
```

File: code/modules/mob/living/carbon/human/species.dm [hunk 2]
```diff
@@ -1931,7 +1931,7 @@ GLOBAL_LIST_EMPTY(roundstart_race_names)
 				if(BP.receive_damage(damage_amount, 0, wound_bonus = wound_bonus, bare_wound_bonus = bare_wound_bonus, sharpness = sharpness))
 					H.update_damage_overlays()
 					if(HAS_TRAIT(H, TRAIT_MASO) && prob(damage_amount))
-						H.mob_climax(forced_climax=TRUE)
+						H.mob_climax(forced_climax=TRUE, "masochism")
```
Comment (LetterN) [inline code/modules/mob/living/carbon/human/species.dm]:
```suggestion
						H.mob_climax(forced_climax=TRUE, cause  = "masochism")
```

File: code/modules/vending/kinkmate.dm [hunk 3]
```diff
@@ -31,6 +31,8 @@
 				/obj/item/autosurgeon/testicles = 3,
 				/obj/item/storage/pill_bottle/penis_enlargement = 5,
 				/obj/item/storage/pill_bottle/breast_enlargement = 5,
+				/obj/item/reagent_containers/glass/bottle/crocin = 5,
+				/obj/item/reagent_containers/glass/bottle/camphor = 5
```
Comment (LetterN) [inline code/modules/vending/kinkmate.dm]:
```suggestion
				/obj/item/reagent_containers/glass/bottle/camphor = 5,
```

File: modular_citadel/code/datums/status_effects/chems.dm [hunk 4]
```diff
@@ -500,13 +500,19 @@
...
 				C.jitteriness += 100
 				C.stuttering += 25
 				C.DefaultCombatKnockdown(60)
 				C.Stun(60)
 				to_chat(owner, "<span class='warning'><i>Your muscles seize up, then start spasming wildy!</i></span>")
 
+			else if (lewd && lowertext(customTriggers[trigger]) == "cum")//aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
+				if(ishuman(C))
+					var/mob/living/carbon/human/H = C
+					H.mob_climax(forced_climax=TRUE, "MKUltra")
```
Comment (LetterN) [inline modular_citadel/code/datums/status_effects/chems.dm]:
```suggestion
					H.mob_climax(forced_climax=TRUE, cause = "MKUltra")
```

**Review by LetterN — CHANGES_REQUESTED** [review]

**Comment by BlueWildrose** [comment]
Oh shit.

Is arousal circuits gonna make a comeback too?

**Comment by Putnam3145** [comment]
> 
> 
> Oh shit.
> 
> Is arousal circuits gonna make a comeback too?

almost definitely not tbh. like, *maybe*? but they're... really extreme even for this stuff, cause they're too easy to target directly without causing loud text that gives you seconds to react.

**Comment by dzahlus** [comment]
my balls finally feel full again

**Comment by keronshb** [comment]
Putnam I sent the station nuke to the ghost cafe to get rid of the horny and this is what I get?

**Comment by SmArtKar** [comment]
we NEED arousal circutry back. Just for memes. Just for lewd

**Comment by silicons** [comment]
Arousal circuits aren't coming back.

**Pull request merged**